SIHRDmodel with variability.Rmd

```diff
@@ -0,0 +1,260 @@
+# model.confirm(data)
+# model.plot(data)
+# model.info(data)
+SIRHD.model = function(N, beta, I.0, p, epsilon, theta, gamma, TimeH, TimeNH){
+  param = c(N, beta/N, epsilon, theta, gamma, TimeH, TimeNH)
+  data = model.data(N, beta, I.0, p, epsilon, theta, gamma, TimeH, TimeNH)
+  plot = model.plot(data)
+  result_info = model.info(data)
+  info = c(param, result_info)
+  return(list(data,plot,info))
+}
+test.model = SIRHD.model(10000, 2, 5, 0.5, 0.8, 0.9, 0.9, 10, 5)
+
+```
+
+###Factorial Design of 5 factors with 288 levels
+
+\begin{itemize}
+    \item Response Variables
+    \begin{itemize}
+        \item Total Death
+        \item Total Infected
+        \item Number of Days (disease lasts)
+        \item Peak Prevalence (Max Infected Proportion)
+        \item Max Hospitalized Proportion
+    \end{itemize}
+    \item Explanatory Variables
+    \begin{itemize}
+        \item Population $N$ (3 levels): small town (10,000), big town (100,000), and mega city (1,000,000)
+        \item Transmission Rate $\beta$ (2 levels): 2/population and 4/population
+        \item Hospitalization Rate $\varepsilon$ (3 levels): 50\%, 60\%, and 70\%
+        \item Recovery Rate with Hospitalization $\theta$ (2 levels): 90\% and 95\%
+        \item Recovery Rate without Hospitalization $\gamma$ (2 levels): 90\% and 95\%
+        \item Average days in hospital $\inv{\psi}$ (2 levels): 10 and 15
+        \item Average days in infected status $\inv{\lambda}$ (2 levels): 5 and 10
+    \end{itemize}
+\end{itemize}
+
+```{r}
+I0 = 5
+S.rate = 0.5
+# List of Explanatory Variables:
+Pop.Lst = c(10000, 100000, 1000000)
+Beta.Lst = c(2,4)
+Hrate.Lst = c(0.5, 0.6, 0.7)
+RrateH.Lst = c(0.9, 0.95)
+RrateNH.Lst = c(0.9, 0.95)
+TimeH.Lst = c(10, 15)
+TimeNH.Lst = c(5, 10)
+#Pop.Lst, Beta.Lst, Hrate.Lst, RrateH.Lst, RrateNH.Lst, TimeH.Lst, TimeNH.Lst
+result_table = data.frame()
+for(N in Pop.Lst){
+  for(b in Beta.Lst){
+    for(hr in Hrate.Lst){
+      for(rrh in RrateH.Lst){
+        for(rrnh in RrateNH.Lst){
+          for(th in TimeH.Lst){
+            for(tnh in TimeNH.Lst){
+              model = SIRHD.model(N, b, I0, S.rate, hr, rrh, rrnh, th, tnh)
+              result_table = rbind(result_table, model[[3]], stringsAsFactors = FALSE)
+            }
+          }
+        }
+      }
+    }
+  }
+}
+colnames(result_table) = c("Population", "TransmissionRate", "HospitalizationRate", "HospitalizedRecoveryRate", "Non-HospitalizedRecoveryRate", "HospitalizationTime", "InfectionTime", "LastDay", "Status", "TotalDeath", "Total Infected", "Max New Case", "PeakDayNewCase", "MaxInHospital", "PeakDayHospital")
+result_table[1:7] = as.data.frame(sapply(result_table[1:7], as.numeric))
+result_table[1:7] = format(result_table[1:7], scientific = FALSE)
+result_table[c(1:7,9)] = as.data.frame(sapply(result_table[c(1:7,9)], as.factor))
+result_table[c(8,10:15)] = as.data.frame(sapply(result_table[c(8,10:15)], as.numeric))
+```
+
+###Factorial Design of 5 factors with 288 levels
+

```

Comment:
Is this a duplicate by mistake or is there anything changed in the previous experiment? (line 150-288)